test/agent.js

```diff
@@ -224,6 +224,41 @@ test('#currentTraceparent', function (t) {
   })
 })
 
+test('#currentTraceIds', function (t) {
+  t.test('no active transaction or span', function (t) {
```

Comment:
It would probably be a good idea to also check the return value of `agent.currentTraceIds.toString()` in this and the two other tests below

Reply:
fixed